x-pack/plugins/security_solution/server/lib/detection_engine/signals/threshold/bulk_create_threshold_signals.test.ts

```diff
@@ -6,175 +6,13 @@
  */
 
 import { loggingSystemMock } from '../../../../../../../../src/core/server/mocks';
-import { normalizeThresholdField } from '../../../../../common/detection_engine/utils';
-import {
-  Threshold,
-  ThresholdNormalized,
-} from '../../../../../common/detection_engine/schemas/common/schemas';
+import { ThresholdNormalized } from '../../../../../common/detection_engine/schemas/common/schemas';
 import { sampleDocNoSortId, sampleDocSearchResultsNoSortId } from '../__mocks__/es_results';
 import { sampleThresholdSignalHistory } from '../__mocks__/threshold_signal_history.mock';
 import { calculateThresholdSignalUuid } from '../utils';
 import { transformThresholdResultsToEcs } from './bulk_create_threshold_signals';
 
 describe('transformThresholdNormalizedResultsToEcs', () => {
-  it('should return transformed threshold results for pre-7.12 rules', () => {
```

Comment:
Are these tests no longer relevant due to the migration?

Reply:
Yes, we shouldn't need to worry about pre-7.12 rules anymore. Since `transformThresholdResultsToEcs` takes a normalized threshold parameter even before these changes, these tests seem to be more about testing the conversion from `Threshold` to `ThresholdNormalized` rather than testing `transformThresholdResultsToEcs` with different inputs. The conversion has test coverage in `utils.test.ts` so I don't think we're losing significant coverage here.